sqlglot/parser.py

```diff
@@ -6936,6 +6938,17 @@ def _parse_window_spec(self) -> t.Dict[str, t.Optional[str | exp.Expression]]:
             "side": self._match_texts(self.WINDOW_SIDES) and self._prev.text,
         }
 
+    def _parse_window_frame_exclude(self) -> t.Optional[str]:
+        if self._match_text_seq("EXCLUDE"):
+            return (
+                (self._match_text_seq("CURRENT", "ROW") and "CURRENT ROW")
+                or (self._match_text_seq("GROUP") and "GROUP")
+                or (self._match_text_seq("TIES") and "TIES")
+                or (self._match_text_seq("NO", "OTHERS") and "NO OTHERS")
+            )
+
+        return None
```

Comment:
Let's not do it like this. Look into `_parse_var_from_options`, I think the implementation should use it.

Reply:
Done https://github.com/tobymao/sqlglot/pull/5058/files#diff-63eb8dd82d3561bc414e3e13cf59516bd6584c5766a10072157a8a5aee4ad85f